siege_utilities/geo/providers/census_geocoder.py

```
@@ -0,0 +1,397 @@
+        result = cg.onelineaddress(input_addr, returntype="geographies")
+        matches = result.get("result", {}).get("addressMatches", [])
+        if not matches:
+            log_info(f"No match for: {input_addr}")
+            return CensusGeocodeResult(input_address=input_addr)
+
+        parsed = _parse_single_result(matches[0])
+        parsed.input_address = input_addr
+        log_debug(f"Matched: {input_addr} -> {parsed.matched_address}")
+        return parsed
+
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)